<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Автоматизация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="picture">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Задача запущена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Сканируемая папка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="in_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Папка для сохранения результатов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="out_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранять оригинал в подпапке RAW</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="star_btn">
     <property name="text">
      <string>Запустить задачу</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="finish_btn">
     <property name="text">
      <string>Остановить задачу</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
